lib/tasks/schools/daily_regeneration.rake

```diff
@@ -0,0 +1,8 @@
+namespace :school do
+  desc "Schools daily regeneration"
+  task daily_regeneration: :environment do
```

Comment:
I've recently updated all of the other rake tasks that are run by cron so that they:

- log a message with timestamp at start
- log a message with timestamp at end
- include a rescue within loops, so that a failure processing one school does not cause the whole job to fail
- within that rescue log the exception to the log file and Rollbar
- provide the name of the batch process in the Rollbar call

In this case a failure might only be a failure to submit the job for processing, but would be good to catch anything.

Can you have a look at the jobs, e.g. `validate_amr_readings.rake` and follow that pattern?